public/src/lib/utils.test.ts

```
@@ -0,0 +1,18 @@
+import { describe, it, expect } from 'vitest';
+
+// Simple utility function for testing
+export function formatDate(date: Date): string {
+	return date.toLocaleDateString('en-US', {
+		year: 'numeric',
+		month: 'long',
+		day: 'numeric'
+	});
+}
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)